docs/workflow.md

```diff
@@ -0,0 +1,75 @@
+## Release operator
+
+### How does release-operator work?
+Release operator reconciles on release CRs. An example of a valid release looks like this:
+
+```
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  creationTimestamp: null
+  name: v11.3.0
+spec:
+  apps:
+  - name: app-1
+    version: 1.2.1
+  components:
+  - name: ignore-me
+    version: 1.0.0
+  - catalog: my-playground-catalog
+    name: deploy-me
+    reference: 1.0.1-a7663534964e4051d3ed957981c4f7885d60d15f
+    releaseOperatorDeploy: true
+    version: 1.0.1
+  date: "2020-04-27T12:00:00Z"
+  state: active
+```
+
+The only part of the release that this operator cares for is the list of components in the spec. It will make sure to deploy all of the components that have `releaseOperatorDeploy` set to true. So in the example above, it will ignore `ignore-me` and deploy `deploy-me` on the CP.
+
+#### What does deploying mean?
+
+So as you can see from the example above, each component can have a subset of the following fields:
+* `catalog`: which catalog to take the component from? (e.g. control-plane-catalog, control-plane-test-catalog)
```

Comment:
This defaults to `control-plane-catalog`, right? Might be good to mention that this will be set if not specified.

Reply:
Yes, but the defaulting happens on the CRD level and I didn't want to couple the doc with that (it could easily become outdated). As far as release-operator cares, it just gets a catalog and passes it through.